src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-flask_restx.py

```diff
@@ -0,0 +1,13 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2005-2020, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License (version 2
+# or later) with exception for distributing the bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#
+# SPDX-License-Identifier: (GPL-2.0-or-later WITH Bootloader-exception)
+# -----------------------------------------------------------------------------
+from PyInstaller.utils.hooks import collect_data_files
+
+datas = collect_data_files('flask_restx', include_py_files=True)
```

Comment:
The templates are not Python files - can we remove the `include_py_files=True`?. I don't need them we I test it but I've only tried a minimal test. If there are modules getting lost then they should be `hiddenimports`.

Reply:
I've tested it without `include_py_files` and checked out `flask-restx` code-base. I don't think we'll miss anything there, I've just removed it.